ingest-app/docs/api.md

```diff
@@ -546,6 +549,62 @@ curl -i -X DELETE \
 {"concept-id":"S1200000015-PROV1","revision-id":2}
 ```
 
+### <a name="create-update-subscription"></a> Create / Update a Subscription
+
+Subscription concept can be created or updated by sending an HTTP PUT with the metadata to the URL `%CMR-ENDPOINT%/providers/<provider-id>/subscriptions/<native-id>`. The response will include the [concept id](#concept-id) and the [revision id](#revision-id).
```

Comment:
Explaining what the `concept-id` and `revision-id` are for to give people some context, e.g. it only applies to collection concepts.